common/src/main/java/com/wynntils/features/alwayson/FixCastingSpellsFromInventoryFeature.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.alwayson;
+
+import com.wynntils.core.features.StateManagedFeature;
+import com.wynntils.mc.event.ArmSwingEvent;
+import net.minecraft.world.InteractionHand;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+/**
+ * This feature fixes an issue where a Archers holding bows may start
+ * casting spells just by dropping items from the inventory screen.
+ */
+public class FixCastingSpellsFromInventoryFeature extends StateManagedFeature {
```

Comment:
We can make this an UserFeature. I get there is no point in disabling this, but StateManagedFeatures are reserved for edge cases in Artemis (and likely will be removed). 

Overall, imo it is fine to give the user the option to disable this.